src/common/NavbarItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function NavbarItem(props) {
+    const history = useHistory();
+    const url = props.title.toLowerCase();
+    return <div className="navbar-list-item" onClick={e => history.push(`/${url}`)}>
+        <div className="navbar-list-item-logo">
+            <img src={props.svg} alt="logo" />
```

Comment:
Why did you choose to implement it this way and not use svg as component instead of passing as prop?